views/inbox/compose.tt

```diff
@@ -0,0 +1,76 @@
+[%- CALL dw.active_resource_group("foundation") -%]
+
+[% dw.need_res({ group => "foundation"}, 
+    'stc/css/pages/inbox.css'
```

Comment:
I'm vaguely surprised this list of files doesn't need trailing commas?

nit: using single quotes for some lines but double quotes for others kills the Mark